src/app/icd/ICDManager.h

```diff
@@ -131,26 +121,16 @@ class ICDManager : public ICDListener
     uint8_t mCheckInRequestCount      = 0;
 
 private:
-    // SIT ICDs should have a SlowPollingThreshold shorter than or equal to 15s (spec 9.16.1.5)
-    static constexpr System::Clock::Milliseconds32 kSITPollingThreshold = System::Clock::Milliseconds32(15000);
-    static constexpr System::Clock::Milliseconds32 kSlowPollingInterval = CHIP_DEVICE_CONFIG_ICD_SLOW_POLL_INTERVAL;
-    static constexpr System::Clock::Milliseconds32 kFastPollingInterval = CHIP_DEVICE_CONFIG_ICD_FAST_POLL_INTERVAL;
-
-    // Minimal constraint value of the the ICD attributes.
-    static constexpr uint32_t kMinIdleModeDuration    = 500;
-    static constexpr uint32_t kMinActiveModeDuration  = 300;
-    static constexpr uint16_t kMinActiveModeThreshold = 300;
-
     KeepActiveFlags mKeepActiveFlags{ 0 };
 
     // Initialize mOperationalState to ActiveMode so the init sequence at bootup triggers the IdleMode behaviour first.
     OperationalState mOperationalState             = OperationalState::ActiveMode;
-    ICDMode mICDMode                               = ICDMode::SIT;
     PersistentStorageDelegate * mStorage           = nullptr;
     FabricTable * mFabricTable                     = nullptr;
     bool mTransitionToIdleCalled                   = false;
     Crypto::SymmetricKeystore * mSymmetricKeystore = nullptr;
     ObjectPool<ObserverPointer, CHIP_CONFIG_ICD_OBSERVERS_POOL_SIZE> mStateObserverPool;
+    ICDData * mICDData = nullptr;
```

Comment:
can you move this up (where ICDMode was) so it aligns with other pointers declarations :) OCD

Reply:
done